process/tags_v2_test.go

```diff
@@ -37,6 +37,18 @@ func TestUnsafeIterationV2(t *testing.T) {
 	})
 }
 
+func FuzzIterateV2(f *testing.F) {
+	f.Add(make([]byte, 6), 1)
```

Comment:
can we give FuzzingIterateV2 a payload that actually has data that can be iterated ? I suspect the fuzzer cannot find new test cases that test the whole function.